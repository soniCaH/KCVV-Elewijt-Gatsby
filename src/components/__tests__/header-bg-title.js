import React from "react"
import renderer from "react-test-renderer"
import HeaderBgTitle from "../header-bg-title"

describe("SimpleTitle", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<HeaderBgTitle title="KCVV Elewijt" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe("HighlightTitle", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<HeaderBgTitle title="KCVV Elewijt" highlight="Er is maar één plezante compagnie" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe("ClassnameTitle", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<HeaderBgTitle title="KCVV Elewijt" classes="title--large" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
