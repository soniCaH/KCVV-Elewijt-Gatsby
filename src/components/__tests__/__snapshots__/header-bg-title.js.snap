// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClassnameTitle renders correctly 1`] = `
<header
  className="page-header-title title--large"
>
  <h1>
    KCVV Elewijt
     
  </h1>
</header>
`;

exports[`HighlightTitle renders correctly 1`] = `
<header
  className="page-header-title "
>
  <h1>
    KCVV Elewijt
     
    <span
      className="highlight"
    >
      Er is maar één plezante compagnie
    </span>
  </h1>
</header>
`;

exports[`SimpleTitle renders correctly 1`] = `
<header
  className="page-header-title "
>
  <h1>
    KCVV Elewijt
     
  </h1>
</header>
`;
